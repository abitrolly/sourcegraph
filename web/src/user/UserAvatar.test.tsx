import React from 'react'
import { UserAvatar } from './UserAvatar'
import { mount } from 'enzyme'

describe('UserAvatar', () => {
    test('no avatar URL', () => expect(mount(<UserAvatar user={{ avatarURL: null }} />)).toMatchSnapshot())

    test('with avatar URL', () => expect(mount(<UserAvatar user={{ avatarURL: 'u' }} />)).toMatchSnapshot())
})
