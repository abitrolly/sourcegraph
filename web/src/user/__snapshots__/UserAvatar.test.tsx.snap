// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserAvatar no avatar URL 1`] = `
<UserAvatar
  user={
    Object {
      "avatarURL": null,
    }
  }
/>
`;

exports[`UserAvatar with avatar URL 1`] = `
<UserAvatar
  user={
    Object {
      "avatarURL": "u",
    }
  }
>
  <img
    className="user-avatar "
    src="u"
  />
</UserAvatar>
`;
